Wikipedia/Code/NSUserDefaults+WMFExtensions.swift

```diff
@@ -116,10 +116,17 @@ let WMFSendUsageReports = "WMFSendUsageReports"
     
     @objc var themeAnalyticsName: String {
         let name = string(forKey: WMFAppThemeName)
+        let darkMode =  bool(forKey: "SystemDarkMode")
         guard name != nil, name != Theme.defaultThemeName else {
+            let theme = Theme.defaultAnalyticsThemeName
+            return darkMode ? Theme.dark.name : theme
```

Comment:
Actually for the system dark mode, we automatically choose our app's "black" theme, not the app's "dark" theme (confusing, I know). Can you send `Theme.black` here instead of `Theme.dark`? It also might help to rename the local `darkMode` var to `systemDarkMode` to maybe help a little with code confusion.